entitiy(회원, 장소, 세부사항, 메뉴, 평점, 카페, 놀거리, 음식점) 구현

sunghun0917 opened the pull request.

hakmini committed:
setting

SanghJung committed:
Merge pull request #5 from SanghJung/user

setting

Commit:
entitiy, memberDto, denpendency(jwt, validation) 추가, yml 파일에서 jpa, logging 설정, securityConfig에서 BCryptPasswordEncoder @Bean 으로 등록

Commit:
entitiy, memberDto, denpendency(jwt, validation) 추가, yml 파일에서 jpa, logging 설정, securityConfig에서 BCryptPasswordEncoder @bean 으로 등록

sunghun0917 commented:
최신으로 commit한 72fa113이 최종입니다.

SanghJung reviewed (changes requested):
메인이 아닌 디벨롭으로 요청해주세요

samdaseuss reviewed (changes requested):
잘못봐서 승인했네요.. main 으로 머지 요청이 잘못되어있습니다

Commit:
entitiy, memberDto, denpendency(jwt, validation) 추가, yml 파일에서 jpa, logging 설정, securityConfig에서 BCryptPasswordEncoder @bean 으로 등록

SanghJung committed:
Merge pull request #13 from SanghJung/entity

Entitiy(회원, 장소, 세부사항, 메뉴, 평점, 카페, 놀거리, 음식점) 구현

SanghJung committed:
user

SanghJung committed:
conflict resolve

SanghJung committed:
Merge pull request #15 from SanghJung/user-service

user

Commit:
yml파일 분리

Commit:
kakao-api 연결

hakmini committed:
login

Commit:
API요청 데이터 DB 저장

Commit:
오류 수정

sunghun0917 committed:
Merge branch 'develop' into entity

SanghJung committed:
Merge pull request #18 from SanghJung/entity

Api요청 및 DB저장

Commit:
place 엔티티 수정

samdaseuss committed:
chore: Add test code configuration

* 테스트 코드를 위한 설정 수행

samdaseuss committed:
style: Update code

samdaseuss committed:
feat: Implement CreateCourse Page

* 코스 이름 생성 페이지 구현

Resolves: #20
Related to: #20

samdaseuss committed:
test: Add CreateCourse test code

코스 이름 저장 화면에 대한 테스트 코드 작성 및 3 Pass 완료
1. 코스 생성 컴포넌트가 렌더링 되는지 여부 확인
2. 코스명을 입력하지 않고 다음 버튼을 클릭했을 시 에러 메시지가 잘 노출되는지 여부 확인
3. 코스명 입력 후 '다음'버튼 클릭 시 페이지가 잘 이동되는지 여부 확인

Resolves: #20
Related to: #20

hakmini committed:
init git

samdaseuss committed:
refactor: code refactoring

Resolves: #20
Related to: #20

samdaseuss committed:
chore: Add dependencies

samdaseuss committed:
feat: Implement AddVisitPurpose page

Resolves: #20
Related to: #20

samdaseuss committed:
test: Add AddDescription page test code

Resolves: #20
Related to: #20

samdaseuss committed:
feat: Add AddDescription page

Resolves: #20
Related to: #20

samdaseuss committed:
chore: Add dependencies

samdaseuss committed:
feat: Implement CreateCourseEnd page

Resolves: #20
Related to: #20

samdaseuss committed:
remove: Delete CreateCourseEnd.js file

Commit:
place 중복 처리

Commit:
Merge branch 'entity' of https://github.com/SanghJung/Capstone-Design into entity

hakmini committed:
20240428

hakmini committed:
20240428

Commit:
충돌 해결

Commit:
출돌 해결

Commit:
entity 추가

SanghJung committed:
Merge pull request #24 from SanghJung/entity

Entity 추가 및 충돌 해결

Commit:
생성자 추가

hakmini committed:
20240501

Commit:
securityConfig 수정

samdaseuss committed:
refactor: 코드 리펙토링

samdaseuss committed:
Merge pull request #27 from SanghJung/create-course

마이 코스 저장 기능 개발 완료

hakmini committed:
20240504

SanghJung committed:
Merge pull request #29 from SanghJung/user-service

User service

Commit:
merge
